packages/ds-healthcare-gov/src/components/Footer/Footer.tsx

```diff
@@ -29,6 +29,8 @@ export interface FooterProps {
    * Element to be added to display content for top footer section changes
    */
   footerTop?: React.ReactNode;
+  logoId?: string;
```

Comment:
is `logoId` being used somewhere?

Reply:
Good catch. I missed this and should have removed it.